lua/precognition/init.lua

```diff
@@ -27,6 +27,7 @@ local M = {}
 ---@field highlightColor vim.api.keyset.highlight
 ---@field hints Precognition.HintConfig
 ---@field gutterHints Precognition.GutterHintConfig
+---@field disabled_fts table
```

Comment:
```suggestion
---@field disabled_fts string[]
```

Reply:
Ah. I wasn't sure what the type should've been. It's my first time working with type hints in Lua, so thanks for the correction!